parallel-consumer-core/pom.xml

```diff
@@ -35,6 +35,12 @@
             <version>1.1.8.4</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
Quick comment - can you make this without adding dependencies?

Reply:
Fixed